docs/components/card-catalog.md

```diff
@@ -470,33 +446,58 @@ This should be used instead of the `items` property.
       :item="item"
       class="catalog-item"
     />
-    <KCatalogItem
-      :item="longItem"
-      :truncate="true"
-      class="catalog-item"
-    />
-    <KCatalogItem>
-      <template v-slot:cardTitle>
-        <h4 class="d-flex">
-          <KIcon
-            icon="profile"
-            color="#7F01FE"
-            size="24" />
-          <span class="ml-2">Call Me!</span>
-        </h4>
-      </template>
-      <template v-slot:cardBody>
-        <span class="mr-2">Take action!</span>
-        <KButton size="small">
-          <KIcon
-            icon="gearFilled"
-            size="16"
-            view-box="0 0 16 16"
-            class="pr-0"
-          />
-        </KButton>
-      </template>
-    </KCatalogItem>
+  </template>
+</KCardCatalog>
+```
+
+If used in conjuction with a `fetcher` you have the option of using the returned `data`.
+
+<KCardCatalog :fetcher="fetcherSm" title="Customized body">
+  <template v-slot:body="{ data }">
+    <div v-for="item in data">
+      <h4>{{ item.title }}</h4>
+      <p>{{ item.description }}</p>
+    </div>
+  </template>
+</KCardCatalog>
+
+```vue
+<KCardCatalog :fetcher="fetcher" title="Customized body">
+  <template v-slot:body="{ data }">
+    <div v-for="item in data">
+      <h4>{{ item.title }}</h4>
+      <p>{{ item.description }}</p>
+    </div>
+  </template>
+</KCardCatalog>
+```
```

Comment:
The pagination in this section in the docs appears to be broken. I can paginate to the second page even though there are only 6 items.

Reply:
Fixed, the initial page size wasn't getting set correctly.